### DISCUSSION TITLE
1.5.2 Release of Kade Engine

### DISCUSSION THREAD

**Discussion by puyoxyz** [KadeArchive/Kade-Engine#815] [General]
You can view the full list of changes [here](https://kadedev.github.io/Kade-Engine/changelogs/changelog-1.5.2)!

<hr /><em>This discussion was created from the release <a href='https://github.com/KadeDev/Kade-Engine/releases/tag/1.5.2'>1.5.2 Release of Kade Engine</a>.</em>

**Comment by DatGamerYoloYT** [comment]
Why is this the source code?

**Reply by MoezTrigui** [reply]
becuse this engine is for mods idiot

**Reply by DatGamerYoloYT** [reply]
It's fixed now, but how can you play a mod when its the source code? and I know its for mods

**Reply by Hythrex** [reply]
you compile it

**Reply by NotIlulus** [reply]
i got big problem it always says obj\msvc19-arm64-nc\70e38311_Context3DElementType.obj : fatal error LNK1112: module machine type 'x64' conflicts with target machine type 'ARM64' no matter what version

**Comment by LinkGoesBowling** [comment]
How do I download this it's just source code and there is no actual game in it and when I found out how to actually download it it just downloads 1.5.1

**Reply by puyoxyz** [reply]
it's fixed now

**Reply by DatGamerYoloYT** [reply]
thanks @daniel11420

**Comment by rednoobguy26** [comment]
how do i get kade engine on mac lolllllllllllll

**Reply by puyoxyz** [reply]
sorry !!! i broke mac builds by accident then fell asleep by accident!!! working on fixing it now!!

**Reply by isakube** [reply]
classic puyo

**Comment by notiriscent** [comment]
awsm pogu

**Comment by c0deblox** [comment]
Poggers.

**Comment by MoezTrigui** [comment]
hey i was working on a mod and i got the update thing can i update it without redoing the code

**Reply by Sentient-Sandbag** [reply]
you cant but you can just copy the old one to the new one

**Comment by D1g1talDev** [comment]
so 1.6 coming in ?  because 1.5 released after 1.4.2 i just want icon grid colors with the health bar  and winning icons and middle scroll and resolution setting, and disable screen moving with the beat and do beat sync with the boyfriend and the oppent

**Reply by isakube** [reply]
PLEEEEEEEEEEEASE LEARN WHAT A FULL STOP IS

**Comment by dattoastguy** [comment]
how do i download for mac

**Comment by NightSalt** [comment]
Why can't I download this???

**Reply by NightSalt** [reply]
nvm I did
